src/main/views/helpers/mousedraghandler.js

```diff
@@ -64,7 +65,12 @@ define(function() {
       return;
     }
 
-    this._mouseDownClientX = event.evt.clientX;
+    if (event.type === 'touchstart') {
+      this._mouseDownClientX = parseInt(event.evt.touches[0].clientX, 10);
```

Comment:
As `clientX` is already a number, I suggest using `Math.floor()` rather than `parseInt()`.

Reply:
Ok, thank you. I changed it.